exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -167,10 +167,12 @@ drill.exec: {
     debug: true,
     janino_maxsize: 262144,
     cache_max_size: 1000,
-    // Enable to write generated source to disk. See ClassBuilder
-    save_source: false,
     // Where to save the generated source. See ClassBuilder
     code_dir: "/tmp/drill/codegen"
```

Comment:
Writing to an external directory may require to add such source directory to the IDE project each time when people try to debug the generated code. 

One way I tried is to put the generated source under $drill-java-exec/target/generated-sources, which is currently for freemarker's generated source.  That way, Intellij is able to load these new generated class automatically when you debug. 

The above approach requires us to exclude those runtime generated class from the distribution jars, by adding the following in java-exec pom.xml. 

```
                  <excludes>
                    <exclude>**/test/generated/**</exclude>
                  </excludes>
``` 

The above comment is optional.